### PR TITLE
Reduced logging and other improvements to console output

### PR DESCRIPTION
**Pull request by kedarbcs16**

### CODE REVIEW

**Comment by Agirish** [comment]
+1. Please run through pre-commit tests and update if no issues.

File: framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java [hunk 1]
```diff
@@ -39,6 +39,8 @@
 import java.util.Random;
 import java.util.concurrent.atomic.AtomicBoolean;
 
+
+
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java]: Why do we have 2 new lines inserted here?

File: framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java [hunk 2]
```diff
@@ -162,10 +165,8 @@ public void run() {
         LOG.info("[" + testStatus + "] (" + stopwatch + ") " + modeler.queryFilename + " (ConnectionID: " + connection.hashCode() + ")");
       }
 
-      if((++countTestsCompleted %100==0 && countTestsCompleted <= totalCases) || (countTestsCompleted == totalCases)){
-        LOG.info(LINE_BREAK);
-        LOG.info("Execution completed for " + countTestsCompleted + " (out of " + totalCases + ") tests");
-        LOG.info(LINE_BREAK);
+      if((++countTestsCompleted %100==0 && countTestsCompleted <= (totalCases*TestDriver.cmdParam.iterations*TestDriver.cmdParam.clones))){
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java]: Isn't countTestsCompleted per iteration? If not it should be. That way we can keep the check as is.

File: framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java [hunk 3]
```diff
@@ -697,6 +698,8 @@ public TestStatus verifyTextPlan(String expectedOutput,
     if (verified) {
       return TestStatus.PASS;
     }
+    actual = actual.substring(0,ALLOWED_ACTUAL_PLAN_VERIFICATION_LENGTH);
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java]: I think we should revert this. We need an intelligent way to crop the text.

File: conf/log4j.properties [hunk 4]
```diff
@@ -1,6 +1,6 @@
 # Root logger option
-log4j.rootLogger=INFO, file, stdout
- 
+log4j.rootLogger=ERROR, file, stdout
+log4j.logger.DrillTestLogger=INFO, Drill
```
Comment (Agirish) [inline conf/log4j.properties]: Curious - why do we have "Drill" here?

File: framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java (hunk 3 above)
Comment (kedarbcs16) [reply framework/src/main/java/org/apache/drill/test/framework/TestVerifier.java]: ok

File: framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java (hunk 2 above)
Comment (kedarbcs16) [reply framework/src/main/java/org/apache/drill/test/framework/DrillTestJdbc.java]: it is not per iteration ,we have to change the logic for it ,this change is due to the fact that it is not an iteration specific thing thus ,the "Execution completed for  ..." is not printed for subsequent iterations

File: framework/src/main/java/org/apache/drill/test/framework/Utils.java [hunk 5]
```diff
@@ -603,14 +603,16 @@ public static String getExistingDrillStoragePlugin(String ipAddress,
...
+    LOG.info("\n>> Path: " + templatePath + "\n");
     File[] templateFiles = new File(templatePath).listFiles();
     for (File templateFile : templateFiles) {
       String filename = templateFile.getName();
       LOG.info(">> Updating File: " + filename);
       String pluginType = filename.substring(0, filename.indexOf('-'));
       boolean isSuccess = Utils.updateDrillStoragePlugin(templateFile.getAbsolutePath(), pluginType, DrillTestDefaults.FS_MODE);
-      LOG.info(">> Update file " + filename + (isSuccess ? " succeeded" : " failed"));
+      if(!isSuccess){
+        LOG.info(">> Update file " + filename + (isSuccess ? " succeeded" : " failed"));
```
Comment (Agirish) [inline framework/src/main/java/org/apache/drill/test/framework/Utils.java]: if !isSuccess is true (upload fails), then there is no need for another check while printing, right?

**Comment by Agirish** [comment]
+1. Thanks for making the changes.

**Pull request merged**